### ISSUE TITLE
"Ready to merge" should work on an approving review

### ISSUE THREAD

**Issue by andrewbranch** [DefinitelyTyped/dt-mergebot#176]
This didn’t trigger a merge: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/46734#pullrequestreview-467443058

**Comment by elibarzilay** [comment]
I don't know if it's a GH bug or just me, but I can't get it to produce the review comments --- running the below (on any PR) doesn't return any comments:

```
query { 
  repository(owner: "DefinitelyTyped", name: "DefinitelyTyped") {
    pullRequest(number: 46734) {
      reviews(last: 100) {
        nodes {
          author { login }
          comments(last: 100) {
            nodes {
              author { login } createdAt body bodyText
            }
          }
        }
      }
    }
  }
}
```

**Comment by andrewbranch** [comment]
This is what you want

```gql
query { 
  repository(owner: "DefinitelyTyped", name: "DefinitelyTyped") {
    pullRequest(number: 46734) {
      reviews(last: 100) {
        nodes {
          author { login }
          bodyText
        }
      }
    }
  }
}
```

**Comment by elibarzilay** [comment]
Bah.  I thought that it appeared in a review comment.

In any case, I'm not sure that it would be a good idea, since it's something that is said before the bot gets a chance to tell you that it's ready to merge.  See for example the case in #164 -- the problem there was that someone said "ready to merge", then there was a discussion and more work, and as soon as it was ready the bot merged it.  With your suggestion, a review of "ready to merge" would (assuming you're the right approver) immediately approve and then merge it, with no chance for any other feedback and getting it to work with a fix for #164 might be tricky.

**Comment by andrewbranch** [comment]
In the case I linked, the bot had already instructed everyone that it could be merged with a “ready to merge” comment. I wouldn’t suggest that a review with that phrase merge the PR unless the PR was already labeled “Merge: auto.”

**Comment by elibarzilay** [comment]
I finally have a fix for #177, which means that in the referenced PR the `Merge:Auto` label that follows your review would also get the "please post a comment" message.  So doing this would be relevant only for cases where someone was confused and made a review instead of a comment.

**Comment by elibarzilay** [comment]
I'm still worried about [the above comment](https://github.com/DefinitelyTyped/dt-mergebot/issues/176#issuecomment-674989884).  Is there anyone else that seconds this feature, or should I close it?
